Readme.md

```diff
@@ -35,10 +37,8 @@ the image directory can be found in the `config.py` file.
 ## Usage
 
 ### Run locally
-
 To run the code without containers, it is sufficient to run 
-separately the `runserver.py` script, and the `worker.py` 
-script. The worker will process jobs stored in the queue. 
+the `start.py` script. 
```

Comment:
What is this script? It was not requested, and it is not listed in the changes. Why do we need it?

Reply:
I added the script to start runserver.py and the workers (worker.py and worker_histo.py) faster.
the idea was to start connection, queue and redis tests as well. No one actually requested it so it will be removed.